types/mydecimal.go

```diff
@@ -2000,6 +2012,7 @@ func DecimalMod(from1, from2, to *MyDecimal) error {
 }
 
 func doDivMod(from1, from2, to, mod *MyDecimal, fracIncr int) error {
+	resetDecimal(mod)
```

Comment:
should we reset `to` here?

Reply:
My mistake, this check now has moved to `DecimalMod`.